include/gsl/span

```diff
@@ -21,13 +21,27 @@
 #include <gsl/gsl_byte>   // for byte
 #include <gsl/gsl_util>   // for narrow_cast, narrow
 
-#include <algorithm> // for lexicographical_compare
-#include <array>     // for array
```

Comment:
Great to see the includes removed. Nice work @JordanMaples!

Reply:
Thanks!